### PR TITLE
Modify somes contents and add files and folders

### PR DESCRIPTION
**Pull request by luisrpavanelli**
Project requirements:

For this project, we added the JavaScript + HTML + CSS linter.

On mobile, we implemented the following features:

- When the user clicks (or taps) the hamburger button, the mobile menu is displayed.
- When the user clicks (or taps) the Close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
mobile menu

- We kept the design as much as possible (e.g. font, colors, images, text, margins) using Figma templates.

- Highlighted the section name in the menu by manipulating the DOM using CSS classes.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+// #Mobile Menu
+const buttonswitch = document.getElementsByClassName('button_switch')[0];
+const wayFinderlines = document.getElementsByClassName('navigation_options')[0];
+const shadowedsurface = document.getElementsByClassName('shadowed_surface')[0];
+const closeMobileButton = document.getElementsByClassName('closemob-table')[0];
+const explorer = document.getElementsByClassName('explorer');
```
Comment (V-Blaze) [inline script.js:6]: - [ ] You have done well implementing the mobile menu, :+1: :+1: But I think there are some issues you need to look into and correct before moving to the next milestone.
- - Currently, clicking on a menu item even when on desktop view opens the mobile menu, this shouldn't be so, as only the hamburger button should open the mobile menu.  I suggest you take a look at this and update your code properly as this doesn't give the best user experience. :+1: :smile: 

- - Video for reference:
https://github.com/luisrpavanelli/Portfolio/assets/107632780/1214b497-e50a-4d03-8585-1554d0f28509

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @luisrpavanelli  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Descriptive pull request summary✔
-  Readme looks profesional👋 
-  Clean organized code✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
project requirements and suggestions, done!

- You have done well implementing the mobile menu, +1 +1 But I think there are some issues you need to look into and correct before moving to the next milestone. OK

- Also I can see that the text on your mobile menu is different from the one in the given design. 100 I know you would like to customize your portfolio, but in this milestone, one of the requirements is to try to stick to the Figma design as much as you can including text, colors etc, I'll suggest you change your text to the ones given in the design. OK

File: style.css [hunk 2]
```diff
@@ -1,772 +1,829 @@
...
 }
 
-.gray-text {
-  color: #6b778c;
+.navigation_options.active {
+  position: absolute;
+  left: 0;
+  padding-top: 6vw;
+  display: flex;
+  font-size: 44px;
```
Comment (GzimAsani) [inline style.css:59]: - [ ] On your last commit you changed the font size from 3.2rem to 44px, the correct one as on the design should be 32px/3.2rem so please update this the previous value. Good job updating the other requests just update this small thing and your project should get approved :+1:

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @luisrpavanelli                 :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with implementing all the requirements :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (luisrpavanelli) [reply style.css]: done!

**Review by binhussen — APPROVED** [review]
Hi @luisrpavanelli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fixed Request Changes ✔️
- Well Organized Code ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @luisrpavanelli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fixed Request Changes ✔️
- Well Organized Code ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**